concrete-core/src/backends/core/implementation/entities/lwe_ciphertext.rs

```diff
@@ -0,0 +1,37 @@
+use super::super::super::private::crypto::lwe::LweCiphertext as ImplLweCiphertext;
+use crate::backends::core::implementation::entities::markers::{CpuStandard32, CpuStandard64};
+use crate::specification::entities::markers::{BinaryKeyFlavor, LweCiphertextKind};
+use crate::specification::entities::{AbstractEntity, LweCiphertextEntity};
+use concrete_commons::parameters::LweDimension;
+
+/// An owned lwe ciphertext in the cpu memory, in the standard domain, using 32-bits precision
```

Comment:
Owned means that the `LweCiphertext32`  owns its internal data (`Vec<u32>` ) and will take care of freeing that memory as opposed to having a reference to some data (eg `&Vec<u32>` or `&[u32]`).

However it may not be necessary to use that word here since it is not mentionned in other types

Reply:
Indeed, it is not necessary. I added this to prepare for the arrival of _view_ types (which do not own the data they point to), but it is too early. Removed.